src/components/next/table/expandable/cell/style.css

```diff
@@ -0,0 +1,38 @@
+@import "../arrow.css";
+
+/* expandable cell */
+:host {
+	display: contents;
+}
+
+:host>div {
+	padding: 0.3rem 1.1em;
+}
+
+:host>div.content {
+	display: flex;
+	align-items: center;
+	white-space: nowrap
+}
+
+:host>div.content:hover {
+	background-color: rgb(var(--smoothly-color-shade));
+}
+
+:host>div:first-child {
+	cursor: pointer;
+}
+
+:host div.detail {
```

Comment:
use `>` in the selector. stencil 4s scoped mode is more open to what you can target and this might cause you to target things that are slotted into you.